packages/apollo-gateway/src/snapshotSerializers/queryPlanSerializer.ts

```diff
@@ -71,23 +84,23 @@ function printNode(
         ) +
         config.spacingOuter +
         indentation +
-        (node.internalFragments.size > 0
-          ? '  ' +
-            Array.from(node.internalFragments)
-              .map(fragment =>
-                printer(
-                  fragment,
-                  config,
-                  indentationNext,
-                  depth,
-                  refs,
-                  printer,
-                ),
-              )
-              .join(`\n${indentationNext}`) +
-            config.spacingOuter +
-            indentation
-          : '') +
+        // (node.internalFragments.size > 0
+        //   ? '  ' +
+        //     Array.from(node.internalFragments)
+        //       .map(fragment =>
+        //         printer(
+        //           fragment,
+        //           config,
+        //           indentationNext,
+        //           depth,
+        //           refs,
+        //           printer,
+        //         ),
+        //       )
+        //       .join(`\n${indentationNext}`) +
+        //     config.spacingOuter +
+        //     indentation
+        //   : '') +
```

Comment:
Curious about the disabling of this and preserving it is intentional? (If so, please add a comment!)